package.json

```diff
@@ -9,9 +9,9 @@
   },
   "scripts": {
     "cover": "nyc --reporter=lcov --reporter=text --reporter=html npm run tests-only",
-    "lint": "eslint --ext .js,.json .",
+    "lint": "eslint .",
     "pretest": "npm run lint",
-    "tests-only": "mocha --recursive",
+    "tests-only": "mocha 'test/bin/**/*.js' 'test/lib/**/*.js' --recursive",
```

Comment:
```suggestion
    "tests-only": "mocha --recursive",
```
i would expect the files to be automatic, or in the config?

Reply:
Under test/fixtures, there are several JavaScript files that we use for testing with ESLint v9, but we are importing a package where they do not exist, so we get an error that the module is not found.

For example, `Cannot find module 'eslint-plugin-plugin'`. These will be modules that do not actually exist because they are mocked up in the test.

Therefore, we explicitly specify the files to be tested.

If you raise mocha to 5.2.0, you can configure the following. (I couldn't get it to work with `--exclude “test/fixtures/**/*.js”` in test/mocha.opts).
![image](https://github.com/user-attachments/assets/a4469d7f-02ba-41dc-b773-9305f2ded514)